core/src/main/java/io/seata/core/serializer/SerializerFactory.java

```diff
@@ -72,4 +72,4 @@ public static Serializer getSerializer(byte serializeCode) {
         return getSerializer(codec).deserialize(bytes);
     }
 
-}
+}
```

Comment:
This kind modification is not necessary.

Reply:
Got it